### PR TITLE
Studies page changes for CHS-Lookit merger - fixes #1197

### LINKED ISSUE #1197: Search page change for CHS-Lookit merger
On narrow screens, studies should appear in a single vertical line, rather than 4 very narrow cards across each row.

### PR DESCRIPTION
**Pull request by becky-gilbert**
Fixes #1197 
This PR makes the "Studies" page more mobile-friendly.
- On narrow/mobile screens, the study list switches from 4 columns of studies to 1 column.
- For the study search/filter/text elements above the study list and navigation, these elements switch from 4 across (wide screen) to 2 columns (medium screens) to 1 column (narrow/mobile).
- Study images now take up the full column width. This makes the image sizing a little more consistent (this is only relevant to older non-square study images that were uploaded before we required them to be square).
- Minor changes to the page's footer to improve appearance on mobile: reduced space above the footer, reduced L/R margin/padding to prevent overlapping elements, and centered the main text.

Mobile, top of Studies page, logged in:

Screenshot 2023-05-04 at 3 19 28 PM

Mobile, top of Studies page, not logged in:

Screenshot 2023-05-04 at 3 20 36 PM

Mobile, middle of Studies page:

Screenshot 2023-05-04 at 3 20 11 PM

Mobile, bottom of Studies page:

Screenshot 2023-05-04 at 3 19 59 PM

Some of the changes above caused very minor changes on wider screens. The things that were affected are: 
- slzing/spacing of the search/filter/text elements above the studies list and navigation
- sizing of study images

Desktop, logged in:

Screenshot 2023-05-04 at 3 18 04 PM

Desktop, not logged in:

Screenshot 2023-05-04 at 3 17 04 PM

And here's the same page on the staging server, for reference:

Screenshot 2023-05-04 at 3 34 39 PM

### CODE REVIEW

**Comment by becky-gilbert** [comment]
Suggestion @mekline - on the mobile version of the "Studies" page, the first study in the list isn't visible without scrolling (see the first 2 screenshots above). I wonder if we could reduce the "Lookit is growing!..." text between the study nav tabs and study list, or show different text on wide vs narrow/mobile screens?

**Comment by becky-gilbert** [comment]
> Suggestion @mekline - on the mobile version of the "Studies" page, the first study in the list isn't visible without scrolling (see the first 2 screenshots above). I wonder if we could reduce the "Lookit is growing!..." text between the study nav tabs and study list, or show different text on wide vs narrow/mobile screens?

@mekline oops, just saw that your list of changes on #1061 [here](https://github.com/lookit/lookit-api/issues/1061#issuecomment-1518172665) includes removing this message! I'll find all of the changes related to the Home page on that comment and update my PR and screenshots.

**Comment by becky-gilbert** [comment]
Here are the changes that I missed from #1061:
- The study location drop-down text and options are now: "Show studies happening..."here on the Lookit platform"/"on other websites"
- Removed the text: "Lookit is growing! We are now showing links to outside studies along with those happening here on Lookit. "

And updated screenshots:

Mobile, not logged in:
Screenshot 2023-05-08 at 2 16 55 PM

Mobile, logged in:
Screenshot 2023-05-08 at 2 17 25 PM

The layout of the search/filter form elements on desktop is now back to looking like it does on production. 
The study images sizing is still modified to be more consistent.

Desktop, not logged in:
Screenshot 2023-05-08 at 2 16 19 PM

Desktop, logged in:
Screenshot 2023-05-08 at 2 16 06 PM

**Comment by becky-gilbert** [comment]
> Great for now! Could you please make a new issue about beautifying/reducing real estate on the mobile view? We should move on for now but remember there is...room for improvement :)

Done! #1219

File: web/templates/web/studies-list.html [hunk 1]
```diff
@@ -48,7 +48,7 @@
                 <a class="text-decoration-none link-dark p-3"
                    href="{% url 'web:study-detail' uuid=study.uuid %}">
                     <img src="{{ study.image.url }}"
-                         class="img-fluid mx-auto d-block"
+                         class="w-100 mw-100 h-auto mx-auto d-block"
                          alt="{{ study.name }}"/>
```
Comment (okaycj) [inline web/templates/web/studies-list.html]: Are both width (w-100) and max-width (mw-100) required here?

**Review by okaycj — APPROVED** [review]
💯 Nice work, I have just two questions.  The first is above and the second... Why lean away from flex?  Do you have a specific reason or was it just easier to use rows?

File: web/templates/web/studies-list.html (hunk 1 above)
Comment (becky-gilbert) [reply web/templates/web/studies-list.html]: Ah, good catch! Nope `mx-100` isn't needed here - I'll remove it.

**Comment by becky-gilbert** [comment]
> Why lean away from flex? Do you have a specific reason or was it just easier to use rows?

Great question. No specific reason. At first I tried to work with flex and couldn't get it to do what I want. Whereas I've been working with the bootstrap grid system a lot lately, so it was easier (for me) to get it working that way. I thought this would be ok since we're using the grid system elsewhere, but I can totally go back to flex if that's the better/preferred way?

**Pull request merged**